services/controllerhost/api_handlers.go

```diff
@@ -109,6 +109,13 @@ func isInputGoingDown(context *Context, hostID string) bool {
 	return state == dfddHostStateGoingDown
 }
 
+// isStoreGoingDown returns true if the specified store host
```

Comment:
can we just make this isHostGoingDown(context *Context, sName, hostID string) rather than having separate ones for different roles?